example/sdf3/sdf3/build.gradle.kts

```diff
@@ -43,10 +42,10 @@ spoofaxLanguageProject {
       builder.addAdditionalCopyResources("target/metaborg/EditorService-pretty.pp.af")
       if(gradle.parent != null && gradle.parent!!.rootProject.name == "devenv") {
         // HACK: use org.metaborggggg groupId for SDF3, as that is used to prevent bootstrapping issues.
-        builder.languageSpecificationDependency(GradleDependency.module("org.metaborggggg:org.metaborg.meta.lang.template:2.6.0-SNAPSHOT"))
+        builder.languageSpecificationDependency(GradleDependency.module("org.metaborg:org.metaborg.meta.lang.template:2.6.0-SNAPSHOT"))
```

Comment:
The dependency with groupId `org.metaborggggg` is there for a reason, please revert.

Reply:
Yes that was unintentionally pushed.